Source/WebCore/rendering/svg/RenderSVGInlineText.cpp

```diff
@@ -89,7 +89,8 @@ String RenderSVGInlineText::originalText() const
 
 void RenderSVGInlineText::setRenderedText(const String& text)
 {
-    RenderText::setRenderedText(text);
+    const auto& textWS = applySVGWhitespaceRules(text, style().whiteSpace() == WhiteSpace::Pre);
```

Comment:
The function returns a copy so this has to be `auto adjustedText =`

Reply:
@smfr - done in https://github.com/WebKit/WebKit/commit/65e9b8c9ee1294f4334e74a34bd52ba75e1d9dd5.

Reply:
@smfr with no warnings or anything?

Reply:
@smfr - I approached it via Blink's patch but I have another patch to fix another test case, which reduces scope of `applySVGWhitespaceRules` as well. So overall `makeString` operations in `applySVGWhitespaceRules` will be halved by another patch.

Reply:
@smfr - based on our discussion, I have updated commit message to reflect on why `whitespace` rules need to be applied before `text-transform` as per web-specification and also other bits.